### ISSUE TITLE
Add support of Symfony 6

### ISSUE THREAD

**Issue by Johnmeurt** [krowinski/php-mysql-replication#92]
Hello,

the library has 2 dependencies on Symfony < 6 which makes the migration impossible for project.
 - `symfony/dependency-injection`: I did not find usage on the source and test. Should it be removed ?
 - `symfony/event-dispatcher`: seems compatible with Sf6.

Regards

**Comment by Moln** [comment]
You can try this fork https://github.com/Moln/php-mysql-replication .

Resolve these issues:

- Add regular expression matching support for `DatabasesOnly` or `TablesOnly` of `Config`.
- Resolve krowinski/php-mysql-replication#94, change static config properties to non-static.
- Add retry feature.
   ```php
   (new ConfigBuilder())
     ->withRetry(-1) // Retry always.
     ->withRetry(0)  // Disable retry feature. (Default)
     ->withRetry(2)  // Retry twice.
   ```
- PHP 8.1,8.2 supports.

**Comment by huangdijia** [comment]
Can you submit a PR to the official?

**Comment by krowinski** [comment]
fixed in #113
